src/packageData.js

```diff
@@ -245,9 +245,39 @@ const packages = Object.assign(
 
 const latestPackages = new Set(Object.values(packages));
 
+const stagePresets = Object.create(null);
+stagePresets[3] = [
+  "@babel/plugin-syntax-dynamic-import",
+  "@babel/plugin-syntax-import-meta",
+  "@babel/plugin-proposal-class-properties",
+  "@babel/plugin-proposal-json-strings",
+];
+stagePresets[2] = [
+  ...stagePresets[3],
+  ["@babel/plugin-proposal-decorators", { "legacy": true }],
+  "@babel/plugin-proposal-function-sent",
+  "@babel/plugin-proposal-export-namespace-from",
```

Comment:
FYI This got moved to a needs consensus but not merged yet: https://github.com/tc39/ecma262/pull/1174